lib/lita/services/market_manager.rb

```diff
@@ -107,8 +107,18 @@ def execute_transaction
           check_limit: false
         )
         return unless karma_transfered
-
-        @lunch_assigner.transfer_lunch(lunch_seller.mention_name, lunch_buyer.mention_name)
+        lunch_transfered = @lunch_assigner.transfer_lunch(lunch_seller.mention_name, lunch_buyer.mention_name)
+        if lunch_transfered
+          sw = Lita::Services::SpreadsheetManager.new(ENV.fetch('GOOGLE_SP_DB_KEY'),'transactions')
+          sw.insert_new_row([
+            Time.now.strftime('%Y-%m-%d'),
```

Comment:
Use 2 spaces for indentation in an array, relative to the first position after the preceding left parenthesis.<br>Do not use Time.now without zone. Use one of Time.zone.now, Time.current, Time.now.in_time_zone, Time.now.utc, Time.now.getlocal, Time.now.xmlschema, Time.now.iso8601, Time.now.jisx0301, Time.now.rfc3339, Time.now.httpdate, Time.now.to_i, Time.now.to_f instead.